test/toporelations.jl

```diff
@@ -445,30 +445,30 @@ end
   elems = connect.([(1, 2, 3), (4, 3, 2)])
   t = HalfEdgeTopology(elems)
   ∂ = Boundary{2,0}(t)
-  @test ∂(1) == (2, 3, 1)
+  @test ∂(1) == (1, 2, 3)
```

Comment:
Element-ordered `halves` results in the head of the first edge matching the original first index of the element.